chainerx_cc/chainerx/native/CMakeLists.txt

```diff
@@ -59,6 +60,12 @@ if(${BLAS_FOUND})
     endif()
 endif()
 
+# LAPACK
+if(${LAPACK_FOUND})
+MESSAGE( STATUS "LAPACK is found, defining CHAINERX_LAPACK_AVAILABLE variable.")
+add_definitions(-DCHAINERX_LAPACK_AVAILABLE=${LAPACK_FOUND})
```

Comment:
Not sure if it's a better name but what do you think of being consistent with the naming conventions according to https://github.com/chainer/chainer/blob/master/chainerx_cc/chainerx/native/CMakeLists.txt#L54.

Reply:
I can change that. I thought it might be better to have a name that describes the situation.